app/src/main/java/com/shagalalab/constitution/data/dao/ArticleDao.kt

```diff
@@ -0,0 +1,12 @@
+package com.shagalalab.constitution.data.dao
+
+import androidx.room.Dao
+import androidx.room.Query
+import com.shagalalab.constitution.data.models.ArticleModel
+
+@Dao
+interface ArticleDao {
+
+    @Query("SELECT * FROM Article")
```

Comment:
I think I know the problem why you are getting 0 result. Seems you haven't tried to open the attached `constitution.db` (all tables names are lowercase there), but you use capital case names everywhere. Changing that should fix your problem.